src/main/src/main/java/org/geoserver/GeoServerNodeData.java

```diff
@@ -0,0 +1,168 @@
+/* (c) 2017 Open Source Geospatial Foundation - all rights reserved
+ */
```

Comment:
Header is usually longer - http://docs.geoserver.org/latest/en/developer/policies/committing.html